Mathlib/Algebra/Group/Pointwise/Set/BigOperators.lean

```diff
@@ -46,7 +46,7 @@ theorem image_finset_prod (f : F) (m : Finset ι) (s : ι → Set α) :
   (image_multiset_prod f _).trans <| congr_arg Multiset.prod <| Multiset.map_map _ _ _
 
 /-- The n-ary version of `Set.mem_mul`. -/
-@[to_additive " The n-ary version of `Set.mem_add`. "]
```

Comment:
I think almost all modified doc-strings in this file should have periods removed: but I'm fine with accepting that as pre-existing